### PR TITLE
feat: add organizations field to support synchronization of multi-organization users

### PR DESCRIPTION
**Pull request by Bsheepcoder**
Fix : #2891

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -30,7 +30,7 @@ require (
 	github.com/go-telegram-bot-api/telegram-bot-api v4.6.4+incompatible
 	github.com/go-webauthn/webauthn v0.6.0
 	github.com/golang-jwt/jwt/v4 v4.5.0
-	github.com/google/uuid v1.4.0
+	github.com/google/uuid v1.6.0
```
Comment (dacongda) [inline go.mod:33]: imho, it is unnecessary to change go.mod , it will be better to open a new pr to upgrade packages

File: web/package.json [hunk 2]
```diff
@@ -23,6 +23,7 @@
     "antd": "5.2.3",
     "antd-token-previewer": "^1.1.0-22",
     "buffer": "^6.0.3",
+    "caniuse-lite": "^1.0.30001621",
```
Comment (dacongda) [inline web/package.json:26]: is this useful?

Comment (Bsheepcoder) [reply web/package.json:26]: Maybe not. I seem to have added it when executing'go mod tidy 'or other commands. I'll pay attention next time.

Comment (hsluoyz) [reply web/package.json:26]: @Bsheepcoder not next time. It needs to be fixed "this time", or this PR will not pass

File: object/user.go [hunk 3]
```diff
@@ -319,6 +320,46 @@ func GetUsers(owner string) ([]*User, error) {
...
+		return nil, fmt.Errorf("The orgs columns should have at least one element")
+	}
+
+	var wg sync.WaitGroup
+	var mu sync.Mutex
+	var allUsers []*User
+	errCh := make(chan error, len(owners))
+
+	for _, owner := range owners {
+		owner := owner // 创建局部变量，避免闭包中的问题
```
Comment (hsluoyz) [inline object/user.go:334]: All English

File: go.mod (hunk 1 above)
Comment (hsluoyz) [reply go.mod:33]: @Bsheepcoder not next time. It needs to be fixed "this time", or this PR will not pass

File: web/src/SyncerEditPage.js [hunk 4]
```diff
@@ -100,6 +106,9 @@ class SyncerEditPage extends React.Component {
 
     const syncer = this.state.syncer;
     syncer[key] = value;
+    if (key === "organization") {
+      syncer.organizations = []; // 清空 organizations 字段
```
Comment (hsluoyz) [inline web/src/SyncerEditPage.js:110]: All English

File: web/yarn.lock [hunk 5]
```diff
@@ -5623,6 +5623,11 @@ caniuse-lite@^1.0.0, caniuse-lite@^1.0.30001464, caniuse-lite@^1.0.30001503:
   resolved "https://registry.yarnpkg.com/caniuse-lite/-/caniuse-lite-1.0.30001507.tgz#fae53f6286e7564783eadea9b447819410a59534"
   integrity sha512-SFpUDoSLCaE5XYL2jfqe9ova/pbQHEmbheDf5r4diNwbAgR3qxM9NQtfsiSscjqoya5K7kFcHPUQ+VsUkIJR4A==
 
+caniuse-lite@^1.0.30001621:
```
Comment (hsluoyz) [inline web/yarn.lock:5626]: not next time. It needs to be fixed "this time", or this PR will not pass

File: web/src/locales/sv/data.json [hunk 6]
```diff
@@ -297,6 +297,7 @@
     "Organization": "Organization",
     "Organization - Tooltip": "Similar to concepts such as tenants or user pools, each user and application belongs to an organization",
     "Organizations": "Organizations",
+    "Organizations - Tooltip": "Provides multi-organization synchronization capabilities",
```
Comment (hsluoyz) [inline web/src/locales/sv/data.json:300]: Just don't update i18n

**Review by hsluoyz — CHANGES_REQUESTED** [review]

**Comment by hsluoyz** [comment]
Closed for now

**Pull request closed without merging**